cmd/main.go

```diff
@@ -1,30 +1,34 @@
 package main
 
 import (
+	"flag"
 	"fmt"
-	"io/ioutil"
-	"log"
-	"net/http"
 	"os"
+	"weather"
 )
 
+type Response struct {
+	OneWord string  `json:"oneword"`
+	Celcius float64 `json:"celcius"`
+}
+
 func main() {
 
+	location := flag.String("location", "", "a city")
```

Comment:
A good way to think about flags is that they're for specifying _options_—that is, things that can be omitted. Can we reasonably omit the location when calling the weather program? Not really! And since users must always enter a location on the command line, there's no real benefit in forcing them to also specify the `-location` flag.

In other words, instead of making them type:

```
./weather -location London
```

let them just type

```
./weather London
```